packages/v4/patternfly-docs/content/design-guidelines/components/spinner/spinner.md

```diff
@@ -11,11 +11,18 @@ Spinners are used to demonstrate data loading. Spinners are offered in different
 
 <img src="./img/spinner-sizes.png" alt="Visual of different spinner sizes" width="436"/>
 
-Use a spinner when the data you are dealing with is unknown in both quantity and shape. Spinners are also good to use when in progress of loading a screen that may not have any data, for example an [empty state screen or failed outcome](/components/empty-state). A common use case for spinners is within a table view, where the spinner can take up the area of the table before it loads.
+### When to use
```

Comment:
I would start the section with "Use a spinner when:" followed by each bullet point that would complete the sentence... same with the "When no to use" section.

So for example
Use a spinner:
- when the data you are dealing with is unknown in both quantity and shape.
- when in progress of loading a screen that may not have any data, for example an [empty state screen or failed outcome](/components/empty-state). 
- within a table view, where the spinner can take up the area of the table before it loads.
- when the expected wait time is between 1-5 seconds.